utils/table-helper.js

```diff
@@ -0,0 +1,11 @@
+const _ = require('lodash');
+
+// NOTICE: Look for the id column in both fields and primary keys
+function hasIdColumn(fields, primaryKeys) {
+  return fields.some(field => field.name === 'id' || field.nameColumn === 'id') ||
+        _.includes(primaryKeys, 'id');
+}
+
+module.exports = {
+  hasIdColumn,
+};
```

Comment:
I think that moving this logic in the `database-analyzer.js` file and passing the `hasIdColumn` info in the `options` result of `analyzeTable` function would really reduce the code size.